### PR TITLE
Events and Competitions section updated to data of 17-18

### PR DESCRIPTION
**Pull request by akshay-99**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1090,186 +1090,179 @@ <h1>
...
     </li>
 
-    <li class="housing handover wow animated fadeInLeft" data-wow-delay=".1s" style="background-size: 55% auto; background-repeat: repeat;" >
+    <li class="housing wow animated fadeInLeft event2" data-wow-delay=".1s">
       <div class="housing-holder">
 
         <div class="description">
           <div class="price-and-rating">
-            <div class="price">12<sup>August</sup></div>
+            <div class="price">26<sup>July</sup></div>
```
Comment (nurdtechie98) [inline index.html]: may be we should use this pattern 26 <sup>th<sup> Jul . seems appropriate

**Comment by nurdtechie98** [comment]
@akshay-99 make these small changes rest looks good to merge

File: index.html (hunk 1 above)
Comment (akshay-99) [reply index.html]: Do you mean this?
screenshot from 2018-06-27 02-11-19


Or should I decrease the font size of the month a bit like this?
screenshot from 2018-06-27 02-25-32


Tell me so I can apply one of these changes to all dates. :v:

Comment (inishchith) [reply index.html]: Former Looks better  . emphasis on [dot] at the end . @nurdtechie98 did you mean that ?

Comment (nurdtechie98) [reply index.html]: @akshay-99 yes, go ahead with the first option

Comment (Freelancer-98) [reply index.html]: Except for the date format, everything seems fine.
Also, that seems to be the wrong picture for BE workshop.
We'll keep you posted about the reports and images for the rest of the stuff.

Comment (akshay-99) [reply index.html]: okay :+1:

Comment (nurdtechie98) [reply index.html]: @Freelancer-98 we don't have any picture of that workshop so this works.

**Comment by akshay-99** [comment]
Done. Changed the date formats.

**Pull request merged**